static/styles/settings.css

```diff
@@ -996,6 +996,18 @@ input[type=checkbox].inline-block {
     border-radius: 0px;
 }
 
+.dropdown-search > input[type=text] {
+    margin: 9px;
+}
+
+.dropdown-list-body {
+    height: auto;
+    max-height: 200px;
+    overflow-y: auto;
+    margin-top: 0;
+    display: block;
+}
```

Comment:
these CSS rules should be encapsulated inside `#id_realm_notifications_stream` so that there's no risk that they impact other parts of the app.